RHEL/7/input/oval/oval_5.11/service_rpcgssd_disabled.xml

```diff
@@ -0,0 +1,42 @@
+<def-group>
+
```

Comment:
@iannko opened a new PR because I was in the midst of deleting rpcgssd
content.

On Mon, Nov 16, 2015 at 9:00 AM, iankko <notifications@github.com> wrote:

> In RHEL/7/input/oval/oval_5.11/service_rpcgssd_disabled.xml
> <https://github.com/OpenSCAP/scap-security-guide/pull/850#discussion_r44941018>
> :
>
> > @@ -0,0 +1,42 @@
> > +<def-group>
> > +
>
> Taking back my previous comment. For the static rpc* NFS services looks
> like their name changed per:
>     https://bugzilla.redhat.com/show_bug.cgi?id=1159183
>
> e.g. nfslock => nfs-lock, rpcgssd => rpc-gssd, rpcsvcgssd => rpc-svcgssd,
> and rpcidmapd => nfs-idmapd in Red Hat Enterprise Linux 7 Update 1 per:
>     https://bugzilla.redhat.com/show_bug.cgi?id=1159183#c0
>
> But this has been since that time recognized to cause compatibility
> issues, and therefore the names have been in recent version of Red Hat
> Enterprise Linux 7 updated the old names to be recognized too. E.g.
>
> service idmapd status
>
> works again in Red Hat Enterprise Linux 7 Update 2 (AFAICT from testing).
> So maybe it's better to keep the old (RHEL-6) names for these services,
> till things will stabilize.
>
> => Ignore my previous comment. Current version is fine (sorry for the
> turmoil).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSCAP/scap-security-guide/pull/850/files#r44941018>
> .
>